spec/multi_patient_api/bulk_export_validation_tester_spec.rb

```diff
@@ -0,0 +1,72 @@
e 'M' = 'Married', given as 'Married'})\\u003c/span\\u003e\\u003c/p\\u003e\\u003cp\\u003e\\u003cb\\u003emultipleBirth\\u003c/b\\u003e: true\\u003c/p\\u003e\\u003ch3\\u003eContacts\\u003c/h3\\u003e\\u003ctable\\u003e\\u003ctr\\u003e\\u003ctd\\u003e-\\u003c/td\\u003e\\u003ctd\\u003e\\u003cb\\u003eRelationship\\u003c/b\\u003e\\u003c/td\\u003e\\u003ctd\\u003e\\u003cb\\u003eName\\u003c/b\\u003e\\u003c/td\\u003e\\u003ctd\\u003e\\u003cb\\u003eTelecom\\u003c/b\\u003e\\u003c/td\\u003e\\u003c/tr\\u003e\\u003ctr\\u003e\\u003ctd\\u003e*\\u003c/td\\u003e\\u003ctd\\u003eEmergency Contact \\u003cspan\\u003e(Details : {http://terminology.hl7.org/CodeSystem/v2-0131 code 'C' = 'Emergency Contact)\\u003c/span\\u003e\\u003c/td\\u003e\\u003ctd\\u003eSarah Abels \\u003c/td\\u003e\\u003ctd\\u003eph: 0690383372(MOBILE)\\u003c/td\\u003e\\u003c/tr\\u003e\\u003c/table\\u003e\\u003ch3\\u003eCommunications\\u003c/h3\\u003e\\u003ctable\\u003e\\u003ctr\\u003e\\u003ctd\\u003e-\\u003c/td\\u003e\\u003ctd\\u003e\\u003cb\\u003eLanguage\\u003c/b\\u003e\\u003c/td\\u003e\\u003ctd\\u003e\\u003cb\\u003ePreferred\\u003c/b\\u003e\\u003c/td\\u003e\\u003c/tr\\u003e\\u003ctr\\u003e\\u003ctd\\u003e*\\u003c/td\\u003e\\u003ctd\\u003eNederlands \\u003cspan\\u003e(Details : {urn:ietf:bcp:47 code 'nl' = 'Dutch', given as 'Dutch'})\\u003c/span\\u003e\\u003c/td\\u003e\\u003ctd\\u003etrue\\u003c/td\\u003e\\u003c/tr\\u003e\\u003c/table\\u003e\\u003cp\\u003e\\u003cb\\u003emanagingOrganization\\u003c/b\\u003e: \\u003ca\\u003eBurgers University Medical Centre\\u003c/a\\u003e\\u003c/p\\u003e\\u003c/div\\u003e\"},\"identifier\":[{\"use\":\"usual\",\"system\":\"urn:oid:2.16.840.1.113883.2.4.6.3\",\"value\":\"738472983\"},{\"use\":\"usual\",\"system\":\"urn:oid:2.16.840.1.113883.2.4.6.3\"}],\"active\":true,\"name\":[{\"use\":\"usual\",\"family\":\"van de Heuvel\",\"given\":[\"Pieter\"],\"suffix\":[\"MSc\"]}],\"telecom\":[{\"system\":\"phone\",\"value\":\"0648352638\",\"use\":\"mobile\"},{\"system\":\"email\",\"value\":\"p.heuvel@gmail.com\",\"use\":\"home\"}],\"gender\":\"male\",\"birthDate\":\"1944-11-17\",\"deceasedBoolean\":false,\"address\":[{\"use\":\"home\",\"line\":[\"Van Egmondkade 23\"],\"city\":\"Amsterdam\",\"postalCode\":\"1024 RJ\",\"country\":\"NLD\"}],\"maritalStatus\":{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v3-MaritalStatus\",\"code\":\"M\",\"display\":\"Married\"}],\"text\":\"Getrouwd\"},\"multipleBirthBoolean\":true,\"contact\":[{\"relationship\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/v2-0131\",\"code\":\"C\"}]}],\"name\":{\"use\":\"usual\",\"family\":\"Abels\",\"given\":[\"Sarah\"]},\"telecom\":[{\"system\":\"phone\",\"value\":\"0690383372\",\"use\":\"mobile\"}]}],\"communication\":[{\"language\":{\"coding\":[{\"system\":\"urn:ietf:bcp:47\",\"code\":\"nl\",\"display\":\"Dutch\"}],\"text\":\"Nederlands\"},\"preferred\":true}],\"managingOrganization\":{\"reference\":\"Organization/f001\",\"display\":\"Burgers University Medical Centre\"}}")
```

Comment:
This should live in a fixture file.

Reply:
Didn't actually use this resource in the tests :/

Reply:
The true power of code reviews!